jvm/src/main/scala/com/intel/oap/extension/ColumnarOverrides.scala

```diff
@@ -316,7 +316,9 @@ case class ColumnarOverrideRules(session: SparkSession) extends ColumnarRule wit
     plan.logicalLink.isDefined
 
   override def preColumnarTransitions: Rule[SparkPlan] = plan => {
-    if (columnarEnabled) {
+    if (nativeEngineEnabled && !plan.isInstanceOf[SerializeFromObjectExec] &&
+      !plan.isInstanceOf[ObjectHashAggregateExec] &&
+      !plan.isInstanceOf[V2CommandExec]) {
```

Comment:
Could you add some explanations on why need to exclude ObjectHashAggregateExec and V2CommandExec here?

Reply:
Added some comments, and create a issue to fix fallback issues: #95